custom-commands/hospitality/deployment/README.md

```diff
@@ -127,11 +127,31 @@ az group delete --name MyResourceGroupName
 
 *This section needs to be updated.*
 
-The resources were created using an azure template which is stored in the [./azuredeploy.json](./azuredeploy.json) file.</br>
-Then the [../storage files](../storage-files) were deployed into the storage resource.</br>
+First an Azure resource group was created using the resourceName parameter as the name.
+
+Then resources were created using an azure template which is stored in the [./azuredeploy.json](./azuredeploy.json) file. Parameters were passed in from the powershell script by manipulation the json values in the [./azuredeploy.parameters.json](./azuredeploy.parameters.json) file. For the individual resources a random number between 0 and 9999 is attached to help guarantee availability.</br>
+
+The values set in the azuredeploy.json file determine many things like pricing scheme, dependencies, and others.
+
+These are the resources specified in the azuredeploy.json:
+* Language Understanding (LUIS)
+* Language Understanding Authoring
+* Speech
+* Storage Account
+* App Service Plan
+* Function App*
+
+\*It is worth noting we set the CORS policy to "\*" from here.
```

Comment:
Don't know what CORs policy is. Is there a link.

Can you put this comment in parenthesis next to Function App instead of footnote?

Reply:
done. Added link to CORS